aten/src/ATen/native/native_functions.yaml

```diff
@@ -771,7 +771,7 @@
   device_check: NoCheck   # TensorIterator
   variants: function, method
 
-- func: bilinear(Tensor input1, Tensor input2, Tensor weight, Tensor? bias) -> Tensor
+- func: bilinear(Tensor input1, Tensor input2, Tensor weight, Tensor? bias=None) -> Tensor
```

Comment:
cc @mruberry is changing this signature BC?

Reply:
I don't think so, no - I think this is an FC issue, however, do you think that's an issue here? Will people begin calling the signature without bias and then old version of PyTorch won't know how to deal with those calls?

Reply:
We're actually having a discussion about this now -- let me follow-up with those teams

Reply:
OK so the way the jit works is that default arguments are not serialized unless they're explicitly specified. So I think this is an FC issue. We could verify by serializing a model with bilinear (without bias called) when PyTorch is built with this commit and then try loading it on PyTorch build before this commit.

Reply:
@mruberry If I split the `=None` change into it's own PR and that gets landed first, would that be okay since the python wrapper will explicitly pass the bias argument?

Reply:
If all the bilinear callsites pass bias explicitly then I think this will be OK from an FC perspective